tchannel-core/src/main/java/com/uber/tchannel/tracing/Tracing.java

```diff
@@ -86,7 +104,7 @@ public static void startOutboundSpan(
                 logger.error("Failed to inject span context into headers", e);
             }
         }
-        outRequest.getFuture().addCallback(new TFutureCallback<Response>() {
+        outRequest.getFuture().addCallback(new TFutureCallback<V>() {
```

Comment:
why `<V>`? It's not used.

Reply:
With the change in line 60 `outRequest.getFuture()` is now parameterized with the `<V>`, hence the callback has to be properly parameterized, too.